addons/udes_stock/views/stock_picking.xml

```diff
@@ -158,6 +158,30 @@
                 <attribute name="attrs">{'readonly': [('state', '!=', 'draft')],  'required': [('picking_type_code', '=', 'incoming')]}</attribute>
             </xpath>
 
+            <xpath expr="//field[@name='date_done']" position="attributes">
+                <attribute name="attrs">{'invisible': [('state', 'not in', ('done', 'cancel'))]}</attribute>
+                <attribute name="string">Date of Transfer</attribute>
+            </xpath>
+
+        </field>
+    </record>
+
+
+    <record id="vpicktree" model="ir.ui.view">
+        <field name="name">stock.picking.tree.udes</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.vpicktree"/>
+        <field name="arch" type="xml">
+
+            <field name="scheduled_date" position="attributes">
+                <attribute name="widget">datetime</attribute>
```

Comment:
Ah looks like it is set to use `remaining_days`, would setting this to blank also have the same change? Just curious.

Reply:
So maybe:

`<attribute name="widget"/>`

Reply:
It was widget="remaining_days" in main form view and that widget shows Today
There are 2 other widgets for date fields as far i noticed "date" and "datetime". In this case it is needed datetime because we want to show hrs too

Reply:
Even that can be as field is datetime , most probably if no widget will show as datetime

Reply:
Yeah so if the default is `datetime`, which it seems it still is, then maybe we can just remove the `widget` attribute to be a tiny bit cleaner. Not a big deal at all though and happy to resolve if you want to keep it explicit.

The main thing is I'm relieved that by default dates and datetimes in the UI will not turn into `Today`, `Tomorrow` etc. :)